ci_framework/roles/libvirt_manager/defaults/main.yml

```diff
@@ -42,8 +42,6 @@ cifmw_libvirt_manager_configuration:
       # GB
       memory: "{{ cifmw_libvirt_manager_compute_memory }}"
       cpus: "{{ cifmw_libvirt_manager_compute_cpus }}"
-      start_ip: 192.168.122.100
-      subnet_mask: 255.255.255.0
```

Comment:
would be nice to have more context about the vars you are removing in the commit message e.g. "vars were deprecated after [1]"

Reply:
I don't even think it was used at some point actually... I just happen to have eyed in that file while doing the README update for the proper structure and saw those 2 params :/.